packages/ui/src/components/page/Sidebar/CurrentMember/CurrentMember.tsx

```diff
@@ -13,16 +13,16 @@ export const CurrentMember = () => {
   const { count, members, active, setActive } = useMyMemberships()
   const [isOpen, toggleOpen] = useToggle()
   const [isCreateOpen, toggleCreateOpen] = useToggle()
+  const forceInitialSelect = () => count > 1 && !active && !isOpen && toggleOpen()
```

Comment:
```suggestion
  const forceInitialSelect = () => {
    if (count > 1 && !active && !isOpen) {
      toggleOpen()
    }
  }
```

Reply:
@p-sad I'd go with something like that. The reason here is that a long construct like this is harder to read later on.

Also being more fluent about the intention makes it is easier to quickly grab what's the function is doing. Especially if there are multiple checks before the function call. 

The `toggleOpen()` call itself is "hidden" behind checks.